packages/default/scss/_variables.scss

```diff
@@ -1148,6 +1148,18 @@ $switch-off-handle-hovered-border: $switch-on-handle-hovered-border !default;
 $switch-off-handle-hovered-gradient: null !default;
 
 
+// Scheduler
+$modal-header-border-color: #cccccc;
+$modal-inner-padding: 1em;
```

Comment:
I see that these two variables are extracted from the `theme` / `layout` files, however, I do not see where are we using them. Could you elaborate on do we need these variables or we can remove them?